pabi_hr_expense_report/pabi_dunning_letter/wizard/pabi_dunning_letter_parser.py

```diff
@@ -3,6 +3,14 @@
 
 
 def pabi_dunning_letter_parser(cr, uid, ids, data, context):
+    if context.get('email_attachment', False):
```

Comment:
Hi Kitti,

We have noticed that when email template execute report action to attach report that time it does not get 'parameters' and data are empty. So we had to use context to pass data...

Regards,
Mustufa